InnerEye/ML/deep_learning_config.py

```diff
@@ -246,8 +256,13 @@ class WorkflowParams(param.Parameterized):
                                 "be relative to the repository root directory.")
 
     def validate(self) -> None:
-        if self.weights_url and self.local_weights_path:
-            raise ValueError("Cannot specify both local_weights_path and weights_url.")
+        if sum([bool(param) for param in [self.weights_url, self.local_weights_path, self.model_id]]) > 1:
```

Comment:
Not worth an extra push, but I think `any([bool(param)...])` would have been clearer

Reply:
The code throws an error if 2 or more options are set, but needs to allow the case were zero or one option is set.